### PR TITLE
Enhancement: Add new icon for worker

### PR DESCRIPTION
**Pull request by dylanbhughes**

### CODE REVIEW

File: src/components/Icon/icons/p-worker.svg
Comment (pleek91) [inline src/components/Icon/icons/p-worker.svg:1]: I think the viewbox for this icon is much smaller than the others. Probably want them to all be the same/similar

https://github.com/PrefectHQ/prefect-design/tree/main/src/components/Icon/icons

Comment (dylanbhughes) [reply src/components/Icon/icons/p-worker.svg:1]: Ah I see, the viewbox for WorkPool is also `viewBox="0 0 24 24"`, let me get back with Karim

Comment (dylanbhughes) [reply src/components/Icon/icons/p-worker.svg:1]: Screenshot 2024-11-06 at 12 30 34 PM
looks like a fair number are off

Comment (dylanbhughes) [reply src/components/Icon/icons/p-worker.svg:1]: @pleek91 Looks like there was a stray `height` and `width` still in the SVG -- fixed!

**Review by pleek91 — APPROVED** [review]

File: src/components/Icon/icons/p-worker.svg
Comment (pleek91) [reply src/components/Icon/icons/p-worker.svg:1]: ah that looks better now! ty

**Pull request merged**